src/main/java/ca/on/oicr/pineryreports/reports/impl/OctaneCountsReport.java

```diff
@@ -173,7 +187,11 @@ public String getTitle() {
   
   @Override
   protected void collectData(PineryClient pinery) throws HttpResponseException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 39 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)